src/main/kotlin/net/aquadc/mike/plugin/android/ReflectPropAnimInspection.kt

```diff
@@ -143,56 +143,56 @@ class ReflectPropAnimInspection : UastInspection(), CleanupLocalInspectionTool {
 
     private companion object {
         private const val AA = "android.animation"
-        private const val TObjectAnimator = "$AA.ObjectAnimator"
-        private const val TPropValsHolder = "$AA.PropertyValuesHolder"
-        private const val TPath = "android.graphics.Path"
-        private const val TView = "android.view.View"
+        private const val TOBJECTANIMATOR = "$AA.ObjectAnimator"
+        private const val TPROPVALSHOLDER = "$AA.PropertyValuesHolder"
+        private const val TPATH = "android.graphics.Path"
+        private const val TVIEW = "android.view.View"
```

Comment:
Should be `SCREAMING_SNAKE_CASE` then (e.g. `T_OBJECT_ANIMATOR`), otherwise unreadable. ButIViolateThisCodestyleRuleWithPleasure